server/asset/btc/rpcclient.go

```diff
@@ -128,10 +131,43 @@ func (rc *RPCClient) GetRawTransactionVerbose(txHash *chainhash.Hash) (*btcjson.
 		true}, res)
 }
 
-// GetBlockVerbose fetches verbose block data for the specified hash.
+// GetBlock fetches raw block data, and the "verbose" block header, for the
+// block with the given hash. The verbose block header return is separate
+// because it contains other useful info like the height and median time that
+// the wire type does not contain.
+func (rc *RPCClient) GetBlock(blockHash *chainhash.Hash) (*wire.MsgBlock, *btcjson.GetBlockHeaderVerboseResult, error) {
```

Comment:
Doesn't need to be exported.

Reply:
Unexported.  Will stay wait for DOGE to go in first as per discussion in https://github.com/decred/dcrdex/pull/1581#discussion_r849513634